plugin/completion.py

```diff
@@ -76,11 +76,13 @@ def format_completion(
         annotation = lsp_label_description or lsp_detail
     elif lsp_label.startswith(lsp_filter_text):
         trigger = lsp_label
-        annotation = lsp_detail
+        if lsp_label_description:
+            annotation = lsp_label_description
+            details.append(html.escape(lsp_detail))
```

Comment:
I think this line should be put under a `if lsp_detail:` condition, because otherwise and if `lsp_detail` is not set by the server, there will be an empty field like `| |` in the details pane, because the items (empty string in this case) get joined together like that (`details=" | ".join(details)`).

Reply:
I've tried to reproduce this case with a test but I can't quite do it.

It's possible that `lsp_detail` is not set by the server but if are gonna just empty string to `details` then `" | ".join(details)` will still result in empty string.

Now this would be problematic with there would be another item added to `details` but here is where this seems like it can't happen. It could only happen if we'd also add another item with:

```
        if lsp_label_detail:
            details.append(html.escape(lsp_label + lsp_label_detail))
```

below but it's not possible to create values that would trigger this case rather than the previous block guarded by  the `if lsp_label_detail and (lsp_label + lsp_label_detail).startswith(lsp_filter_text):` condition.

Reply:
I will still add an if check to not unnecessarily add empty item to `details` and potentially guard future code from triggering this issue.